README.md

```diff
@@ -31,9 +31,9 @@ cd src/java
 |:------------------------|:--------:|:---:|
 |/config-management     | Scripts to deploy the code for the embedded systems.         |  hegyibalint   |
 
-#### Sources divided by programming language
+### Sources divided by programming language
```

Comment:
Please also change the `+ [Sources divided by programming language](#sources-divided-by-programming-language)` in the TOC for `* [Sources divided by programming language](#sources-divided-by-programming-language)`, since the indentation changed and the TOC was 'manually' generated.

Reply:
Fixed